<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintBrushTool</class>
 <widget class="QWidget" name="PaintBrushTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>341</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="eraseHLayout">
    <item>
     <spacer name="eraseHSpacer_left">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="eraseCheckBox">
      <property name="text">
       <string>Erase Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="eraseHSpacer_right">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="activeHLayout">
    <item>
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>Active:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="maskHLayout">
    <item>
     <widget class="QLabel" name="maskLabel">
      <property name="text">
       <string>Mask:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="maskComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>86</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="brushHLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="brushLabel">
      <property name="text">
       <string>Brush Size:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="brushHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="thresholdGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Threshold</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>112</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="lowerHLayout_bottom"/>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>41</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="upperHLayout_bottom"/>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="upperHLayout_top">
     <item>
      <widget class="QLabel" name="upperLabel">
       <property name="text">
        <string>Upper:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="upperHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>91</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="lowerHLayout_top">
     <item>
      <widget class="QLabel" name="lowerLabel">
       <property name="text">
        <string>Lower:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="lowerHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>107</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
